Library/Tales of the Solar Patrol/Races and Animals/Jupiter/The Overlord of Jupiter.gcs

```diff
@@ -4159,13 +4159,5 @@
 			]
 		}
 	],
```

Comment:
Looks like you edited this file by hand, rather than with GCS. This is technically invalid json now, as the trailing comma isn't permitted.